docs/data-sources/slo.md

```diff
@@ -0,0 +1,146 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_slo Data Source - terraform-provider-grafana"
+subcategory: "SLO"
+description: |-
+  Datasource for retrieving all SLOs.
+  Official documentation https://grafana.com/docs/grafana-cloud/slo/API documentation https://grafana.com/docs/grafana-cloud/slo/api/
+---
+
+# grafana_slo (Data Source)
```

Comment:
```suggestion
# grafana_slos (Data Source)
```

I would call it slos because it doesn't read a single SLO like the resource does